src/test/java/plugins/WorkflowPluginTest.java

```diff
@@ -196,30 +201,49 @@
 
     @Category(DockerTest.class)
     @WithDocker
-    @WithPlugins({"workflow-aggregator@1.14", "docker-workflow@1.4", "git", "ssh-agent@1.10"})
+    @WithPlugins({"workflow-cps@2.0", "workflow-job@2.0", "workflow-durable-task-step@2.0", "docker-workflow@1.5", "git", "ssh-agent@1.10", "ssh-slaves@1.11"})
```

Comment:
`workflow-aggregator` pulls in many deps we do not need here, slowing down test initialization